src/index.spec.jsx

```diff
@@ -1,7 +1,13 @@
 import CozyClient from 'cozy-client'
+import jestFetchMock from 'jest-fetch-mock'
+
 import * as cozyBar from './index'
 
 describe('init', () => {
+  beforeAll(() => {
+    global.fetch = jestFetchMock
+  })
```

Comment:
I don't expect to add this in the test. Why?

nit: I was expecting a test to check setWebviewContext is provided as a proxiedApi.

Reply:
Tests were broken without that mock. Not sure what config is the problem but it seemed good enough for now as I'm not updating everything